data/json/recipes/food/bread.json

```diff
@@ -47,5 +47,55 @@
       [ [ "salt", 1 ] ]
     ],
     "//": "Later: this needs baking soda. Also possibility of rewrite to use portions of components' charges."
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "hallula",
```

Comment:
Am I missing something here? These two recipes look identical - the one that supposedly contains milk does *not* include milk in its components.

Reply:
I can't remember why I left this comment justifying the change.

> "//2": "Since milk is only used sparingly in the creation of hallula, to give it a glazed appearance, Cataclysm practicality suggests omitting it."

I think I did this specifically to make a cosmetic distinction between the two, but since I can't for my life understand why I did that, I'll just kill the `without_milk` one.